inclusive-design/wecount.inclusivedesign.ca
# Style page content

## Description

Built mobile and desktop styles for page content.

## Steps to test

1. Run "npm run dev" to start the website and access it at "http://localhost:3000"
2. Page content style should match what's on the design.

**Expected behavior:** 

This should match the [visual design](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=239%3A0).

## Additional information



On line `9-23` of `components/Aside.vue`, `<li> `are hard coded to show an example of how styles will look. This will be replaced by the commented out html code above which generates table of contents dynamically.

## Related issues



resolves #115
